Tests/TuistKitTests/Generator/GeneratorModelLoaderTest.swift

```diff
@@ -278,21 +223,48 @@ class GeneratorModelLoaderTest: XCTestCase {
                                            testAction: testAction,
                                            runAction: runActions)
         // When
-        let model = try TuistKit.Scheme.from(json: try manifest.toJSON())
+        let model = TuistKit.Scheme.from(manifest: manifest)
         
         // Then
         assert(scheme: model, matches: manifest)
     }
     
+    func test_platform_watchOSNotSupported() {
```

Comment:
just spotted we had a discrepancy in supported platforms - I could remove it from `ProjectDescription` to keep it consistent and re-add it once it's supported